_sass/components/_components.textbook__sidebar-right.scss

```diff
@@ -1,11 +1,13 @@
 .sidebar__right {
-    position: absolute;
-    top: 0;
-    right: 0;
+    position: fixed;
+    top: 8%;
+    left: 1030px;

```

Comment:
This means that the sidebar will always be just to the right of the content, provided the screen is wide enough. (as opposed to your original PR, which would put this at the right of the screen regardless of the size). Is that OK?